src/elements/Segment/SegmentGroup.js

```diff
@@ -0,0 +1,42 @@
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+import META from '../../utils/Meta'
+import {
+  getUnhandledProps,
+} from '../../utils/propUtils'
+import _Segment from './_Segment'
+
+function SegmentGroup(props) {
+  const {
+    children, className,
+  } = props
+
+  const classes = cx(
+    className
+  )
+
+  const rest = getUnhandledProps(SegmentGroup, props)
+
+  return (
+    <_Segment className={classes} {...rest} _sdClass='sd-segment-group' _segmentClass='segments'>
+      {children}
+    </_Segment>
```

Comment:
>On another note on your first point, I've pretty much been going on the Stardust docs for figuring out props for each component. These docs say Segment can be raised, piled, stacked etc. I guess these examples need relocated under Segments?

I was almost completely wrong here.  I doubled checked the SUI docs and the individual Segment does support raised, stacked and piled.  Just not horizontal, that appears to only be for groups of segments.

I would make a habit of referencing the official SUI docs (maybe even over the Stardust docs) when determining component APIs.  This will help ensure we cover all the bases.

>The re-structuring you suggested above, has solved this in the case, but doesn't for other similarly nested components. What do you think the best way of remedying this is ?

We're able to teach react-docgen how to resolve doc blocks by reference.  This would allow us to infer docs by component reference.  If possible, I'd like to keep it as straight forward as possible and put the documentation on the component implementing the private component API.